makefile

```diff
@@ -74,7 +74,7 @@ gen-protobuf:
 
 gen-docs:
 	go run ./docs docs
-	gen-readme
+	make gen-readme
```

Comment:
`gen-docs: gen-readme`

then the `gen-docs` target depends on `gen-readme`

Reply:
that would processes the targets in the wrong order. `go run ./docs docs` generates files that are used by `gen-readme`

Reply:
added comment to clarify this